salt/states/group.py

```diff
@@ -83,10 +83,14 @@ def _changes(name,
             ret['comment'] = 'Invalid gid'
             return ret
 
-    if members:
-        # -- if new member list if different than the current
-        if set(lgrp['members']).symmetric_difference(members):
-            change['members'] = members
+    if members is not None:
```

Comment:
Given L87, couldn't this just be `if members:` and then move the else block out to align with this conditional?

Reply:
The first if checks whether any value was supplied to members, the second if checks whether it's an empty list.

I could probably turn it into something like:
```
if members is not None and not members:
    [...]
elif members:
    [...]
```

Would that be preferable?